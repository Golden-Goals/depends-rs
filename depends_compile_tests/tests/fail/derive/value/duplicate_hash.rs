use depends::derives::Value;

#[derive(Value)]
struct Foo<T> {
    bar: Vec<T>,
    #[depends(hash)]
    number: usize,
    #[depends(hash)]
    other: usize,
}

fn main() {}
