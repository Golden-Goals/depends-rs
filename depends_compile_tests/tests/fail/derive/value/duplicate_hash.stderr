error: Attribute specified more than once
 --> tests/fail/derive/value/duplicate_hash.rs:8:15
  |
8 |     #[depends(hash)]
  |               ^^^^
